### DISCUSSION TITLE
Align two float

### DISCUSSION THREAD

**Discussion by hugowaals** [Bodmer/TFT_eSPI#1769] [General]
good day,

I would like to know how to get 2 floats, for example 3.2 and 12.7 with the decimal sign. I've tried this with tft.drawFloat and also by putting them in a String with dtostrf first but I can't get it to work

thank you in advance,
Hugo

**Comment by Bodmer** [comment]
Try this example:
https://github.com/Bodmer/TFT_eSPI/tree/master/examples/320%20x%20240/TFT_Float_Test

**Comment by hugowaals** [comment]
Many thanks again!

 

Best regards,

Hugo

 

 

Van: Bodmer ***@***.***>
Beantwoorden - Aan: Bodmer/TFT_eSPI ***@***.***>
Datum: maandag 11 april 2022 om 18:33
Aan: Bodmer/TFT_eSPI ***@***.***>
CC: "Hugo (Prive)" ***@***.***>, Author ***@***.***>
Onderwerp: Re: [Bodmer/TFT_eSPI] Align two float (Discussion #1769)

 

Try this example:
https://github.com/Bodmer/TFT_eSPI/tree/master/examples/320%20x%20240/TFT_Float_Test

**Comment by hugowaals** [comment]
Dear Bodmer,

 

Do you think this display will work with your TFT-ESPi library?  See attached foto

 

Thanks in advance,

Hugo

 

https://nl.aliexpress.com/item/1005004042373799.html?spm=a2g0o.productlist.main.1.7b7d5df2O6B9V7&algo_pvid=a54f53eb-6276-4471-a1b1-0abf8d477740&algo_exp_id=a54f53eb-6276-4471-a1b1-0abf8d477740-0&pdp_npi=4%40dis%21EUR%2115.27%2111.45%21%21%2116.40%21%21%4021038eda16919581694636052e133f%2112000027832800317%21sea%21NL%21774454616%21&curPageLogUid=CuenKLBrg6Zl
